challenges/6-converter-segundos/C/emptybrain/segundos.c

```diff
@@ -0,0 +1,12 @@
+#include "stdio.h"
+
+int main() {
+  printf("Digite os segundos: ");
+  int seconds;
+  scanf("%d", &seconds);
+  int hour = seconds / 3600;
+  int minutes = (seconds - hour * 3600) / 60;
+  seconds = seconds - hour * 3600 - minutes * 60;
+  printf("%d:%02d:%d \n", hour, minutes, seconds);
```

Comment:
Resolveu o problema dos minutos, mas acredito que é preciso adicionar o mesmo para as horas e segundos:
```suggestion
  printf("%02d:%02d:%02d \n", hour, minutes, seconds);
```